samples/bot-request-approval/nodejs/Readme.md

```diff
@@ -61,7 +61,8 @@ This sample shows a feature where:
 ### 1. Setup for Bot
 In Azure portal, create a [Azure Bot resource](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-authentication?view=azure-bot-service-4.0&tabs=csharp%2Caadv2).
 
-### 2. Run your bot sample
+- Ensure that you've [enabled the Teams Channel](https://docs.microsoft.com/en-us/azure/bot-service/channel-connect-teams?view=azure-bot-service-4.0)### 2. Run your bot sample
```

Comment:
Is this correct here?
"### 2. Run your bot sample"

![image](https://user-images.githubusercontent.com/59052547/147835269-582b09d4-f9aa-4f0a-924f-9e7089dd2809.png)